src/main/java/com/example/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package com.example.bookshop.dto;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull(message = "can't be null")
     private String title;
+    @NotNull(message = "can't be null")
     private String author;
+    @NotNull(message = "can't be null")
```

Comment:
I guess we need also specify in the message, which particular field caused an error, what do you think?